lib/dfe/analytics.rb

```diff
@@ -74,6 +76,22 @@ def self.configure
       config.queue                 ||= :default
     end
 
+    def self.initialize!
+      DfE::Analytics::Fields.check!
+
+      @shown_deprecation_warning = false
+
+      entities_for_analytics.each do |entity|
+        model = model_for_entity(entity)
+        if model.include?(DfE::Analytics::Entities) && !@shown_deprecation_warning
+          Rails.logger.info("DEPRECATION WARNING: DfE::Analytics::Entities was manually included in a model (#{model.name}), but it's included automatically since v1.4. You're running v#{DfE::Analytics::VERSION}. To silence this warning, remove the include from model definitions in app/models.")
+          @shown_deprecation_warning = true
```

Comment:
It's tempting to include a deprecation warning for every model that does this. But, then, if it's only included in `ApplicationRecord` then that might be annoying.

I'd go for annoy and apologise later. 😛

Reply:
I agree, let's be nuisances